A.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="A.css">
+</head>
+<body>
+    <h1>Hello Microverse!</h1>
```

Comment:
- [ ] I think it would be great if you add a class on your `h1` so you can style it be the class instead of HTML tag it is better to follow best practice for CSS and to follow the requirements ☺💪🏻💯🚀

![image](https://user-images.githubusercontent.com/58553711/197898088-7c226ac4-41c8-485b-a8a9-d1511a652372.png)

Reply:
sure i will change that. Thanks!